assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm

```diff
@@ -819,7 +819,6 @@ function printView(url) {
 			}
 		});
 
-		ASN.enableLinks();
```

Comment:
Hi, is this method on [assignments.js](https://github.com/sakaiproject/sakai/blob/07ce149f5081fc26d29dca6a956528e4343901c9/assignment/tool/src/webapp/js/assignments.js) still relevant?

Reply:
It doesn't make any sense to me.